go/vt/vtorc/logic/topology_recovery.go

```diff
@@ -102,30 +102,21 @@ const (
 // TopologyRecovery represents an entry in the topology_recovery table
 type TopologyRecovery struct {
 	ID                     int64
-	UID                    string
 	AnalysisEntry          inst.ReplicationAnalysis
 	SuccessorHostname      string
 	SuccessorPort          int
 	SuccessorAlias         string
-	IsActive               bool
 	IsSuccessful           bool
 	AllErrors              []string
 	RecoveryStartTimestamp string
 	RecoveryEndTimestamp   string
-	ProcessingNodeHostname string
-	ProcessingNodeToken    string
-	Acknowledged           bool
-	AcknowledgedAt         string
-	AcknowledgedBy         string
-	AcknowledgedComment    string
-	LastDetectionID        int64
+	DetectionID            int64
 	RelatedRecoveryID      int64
 	Type                   RecoveryType
 }
 
 func NewTopologyRecovery(replicationAnalysis inst.ReplicationAnalysis) *TopologyRecovery {
 	topologyRecovery := &TopologyRecovery{}
-	topologyRecovery.UID = util.PrettyUniqueToken()
```

Comment:
I think we can remove `PrettyUniqueToken` at this point as well, as nothing calls it anymore.